pkgs/by-name/fr/freeshow/package.nix

```
@@ -0,0 +1,80 @@
+{
+  lib,
+  fetchurl,
+  stdenv,
+  undmg,
+  appimageTools,
+}:
+
+let
+  pname = "freeshow";
+  version = "1.2.1";
+  src =
+    fetchurl
+      {
+        x86_64-darwin = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.dmg";
+          hash = "sha256-OO0uQ6oS1GKiBGz3Qt9jDCY+qdWlgTOY+SiLJB5xQ3c=";
+        };
+        x86_64-linux = {
+          url = "https://github.com/ChurchApps/FreeShow/releases/download/v${version}/FreeShow-${version}.AppImage";
+          hash = "sha256-RBI8IkxY6Xd36vCVDJy9sqpEisB/48hwQo+mg5XTCOs=";
+        };
+      }
+      .${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.system}");
```

It won't evaluate for platforms absent from the list of supported platforms anyway
```suggestion
      .${stdenv.hostPlatform.system};
```

---

It's common, but not necessary. See #314224